app/assets/javascripts/discourse/app/widgets/header.js

```diff
@@ -506,11 +506,27 @@ export default createWidget("header", {
 
     this.state.userVisible = !this.state.userVisible;
     this.toggleBodyScrolling(this.state.userVisible);
+    //
+    // auto focus on first button in dropdown
+    schedule("afterRender", () => {
+      let elem = document.querySelector(".user-menu button");
+      if (elem) {
+        elem.focus();
+      }
```

Comment:
Did we sort out the `?.` operator situation? That's a great place to use it. 😃

Reply:
That shouod be very easy now I guess? just one babel plugin to add

Reply:
It is already supported, the only step where it was previously failing was in minification/compression with `uglifyjs`, but we've switched to `terser` everywhere now.

Reply:
It seems to transpile locally, let's try in CI!

Reply:
Ok I see I added the plugin when I moved to babel 7, but thought I add removed it when we could use minification yet, but we just stopped using it in code, the plugin was still there, cool :+1: